### DISCUSSION TITLE
p207 App.js import {SafeAreaView} 질문 입니다.

### DISCUSSION THREAD

**Discussion by slo-ow** [velopert/dealing-with-react-native#23] [Q&A] [answered]
예제 진행중, useEffect 사용하는 부분에서  (in target 'React-CoreModules' from project 'Pods') 라는 에러를 마주쳤는데 구글링을 통해서 해당 프로젝트에서 pod install 재설치를 통해서 해결을 했습니다. 그리고 그 이후에 p207에 나온 내용처럼 기존의 `import {SafeAreaProvider, SafeAreaView} from 'react-native-safe-area-context';`에서
`import SafeAreaView from 'react-native-safe-area-view';` 으로 변경했더니 status bar 영역이 제대로 동작하지를 않습니다.

<img width="495" alt="스크린샷 2022-02-17 오후 7 53 03" src="https://user-images.githubusercontent.com/61968619/154467447-76e7c818-c5a2-4f72-a2a3-5d3aac22cf70.png">
<img width="428" alt="스크린샷 2022-02-17 오후 7 53 21" src="https://user-images.githubusercontent.com/61968619/154467505-8def4436-04c7-4a99-93fa-4e70701639ae.png">
<img width="539" alt="스크린샷 2022-02-17 오후 7 53 56" src="https://user-images.githubusercontent.com/61968619/154467559-82c49a11-ac12-4296-9cda-28572bf53f7f.png">
<img width="433" alt="스크린샷 2022-02-17 오후 7 54 03" src="https://user-images.githubusercontent.com/61968619/154467577-76a00757-77f1-4399-a5e9-f5f04a181fc2.png">

**Comment by velopert** [comment, marked as answer]
'react-native-safe-area-context' 에서 불러오는 것이 맞습니다.
잘못된 내용이 기재됐네요. 고치도록 하겠습니다. 감사합니다
